frontend/src/components/models.ts

```diff
@@ -2,7 +2,15 @@ import { BigNumber } from 'src/utils/ethers';
 import { JsonRpcSigner, Web3Provider } from 'src/utils/ethers';
```

Comment:
*arbitrary comment location*

I think we need to make updates to the .env temlplate files in both directories, as well as the README file(s)

Reply:
Done: https://github.com/ScopeLift/umbra-protocol/pull/636/commits/edb08fd615f75abf21c03d4c3304582b663534df